src/views/EvaluationList.js

```diff
@@ -0,0 +1,64 @@
+const EvaluationList = function() {
+    this.markup = `
```

Comment:
there is no need for the string template here... the EvaluationPage(GetEvaluationPageData()) already returns a string